etc/services/Step 1 - Collect.cfg

```diff
@@ -53,7 +53,7 @@
d is subscribing to the accelerometer data.\"},{\"position\":1,\"target\":\"CollectRecent\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"Convert those values into a sliding window list. A sliding window list will remove the oldest data point in the list to make room for a new data point.  This is done to ensure a consistent data shape for plotting charts on the UI.\"},{\"position\":1,\"target\":\"Simplify\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"Using the list from the SlidingWindow block, this reduces the overall number of data points in the list while still maintaining the same shape of the plot. It is mainly here for making the chart on the UI look better, you shouldn\u2019t need to change anything in here.\"},{\"position\":1,\"target\":\"JoinFreeDraw\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"This groups all of the incoming signals into one signal prior to being published to the UI.\"},{\"position\":1,\"target\":\"SendFreeDrawUI\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"The publisher block allows you to send data to any participants in the nio system. This could be another service, a separate instance, or a UI. This block sends the consolidated accelerometer data to the UI over the `mobiledemo.freedraw` topic.\"},{\"position\":0,\"target\":null,\"left\":0,\"top\":-300,\"width\":800,\"align\":\"center\",\"content\":\"##Tutorial Introduction\\nWelcome to the \u201ctutorial 1: mobile\u201d system. As a part of this tutorial, we will connect your smartphone to a nio system that contextualizes and visualizes data coming from your phone\u2019s sensors. Let\u2019s get started!\\n\\n* Go to https://mobile.demo.niolabs.com on your mobile browser of choice\\n* Sign into your nio account\\n* Select your mobile tutorial system\u2019s name (Not required if you have only one nio system)\\n* Open https://mobile.demo.niolabs.com in a new tab on your desktop \\n* Select your mobile tutorial system\u2019s name (Not required if you have only one nio system)\\n* Continue the tutorial by following along the service below!\\n\\n####Service Overview\\nThis first service subscribes to the sensor data coming off your phone. The service is fully configured. All you have to do is start it, go check out the desktop UI, and see what happens when you move your phone. For the curious, the dev notes next to the blocks explain how the service is built.\\n\\n\"},{\"position\":0,\"target\":null,\"left\":100,\"top\":900,\"width\":600,\"align\":\"center\",\"content\":\"It\u2019s that easy for you and your phone to join a nio system! This first visualization is messy and contains a lot of raw data. nio can help make sense (and capture value) out of the noise. In the next two services, we will explore different transformations of your phone\u2019s sensor data and teach basic nio service building concepts. Select \u201cService 2\u201d in the service panel to continue...\"}]}",
```

Comment:
I'm not sure how I feel about the centered header in this service, its a little long for that, but content wise it looks legit.